apps/omar-ui-app/grails-app/assets/javascripts/omar/list/list.controller.es6

```diff
@@ -599,7 +635,7 @@
 
     vm.openTab = tab => {
       setTimeout(function() {
-        $(`[data-target="#${tab}"]`).tab("show");
+        //$(`[data-target="#${tab}"]`).tab("show");
```

Comment:
We might want to put this back in.  I know we commented it out during testing.  Let's see if it still works.  Also, is it needed???   Try and find out where it is being used.

Reply:
So I found out this is to determine which tab will be open when you click certain buttons on the cards. For example when you click the toolbox icon on the card it opens up to the tool box tab and when you click the metadata icon it opens up to the metadata tab. Without that no matter what icon you click it will open to the first tab metadata. I uncommented it.